wagtail/images/management/commands/wagtail_update_image_renditions.py

```diff
@@ -0,0 +1,71 @@
+from django.core.management.base import BaseCommand
+
+from wagtail.images import get_image_model
+
+
+class Command(BaseCommand):
+    """Command to remove all generated image renditions."""
+
+    help = "This command will update all image renditions, with an option to purge existing renditions first."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--purge",
+            action="store_true",
+            help="Purge and regenerate all image renditions",
+        )
+        parser.add_argument(
+            "--purge-only",
+            action="store_true",
+            help="Purge all image renditions without regenerating them",
+        )
+
+    def handle(self, *args, **options):
+        renditions = get_image_model().get_rendition_model().objects.all()
```

Comment:
It would be good to return early here if there are no renditions, maybe with a message

`self.stdout.write('No image renditions found')`